blog/index.md

```diff
@@ -33,9 +20,25 @@ layout: default
                     {{ post.date | date: "%b %-d, %Y"}}
                 </span>
             </p>
+
+            {% comment %}
+                http://stackoverflow.com/a/25466298/1475412
+            {% endcomment %}
+            {% assign images = post.content | split:"<img " %}
+            {% for image in images %}
+                {% if image contains 'src' %}
+                    {% assign attributes = image | split:"/>" | first %}
+                    <a href="{{ post.url }}">
+                        <img {{attributes}} />
```

Comment:
I realize now that it would be preferable to pick out only the `src` of the image rather than all of its attributes. However... I have no idea how to do that with Jekyll/Liquid.